heat/core/operations.py

```diff
@@ -321,6 +309,13 @@ def sqrt(x, out=None):
     """
     return __local_operation(torch.sqrt, x, out)
 
+def sum(x, axis=None):
+    # TODO: document me
```

Comment:
Would you be so kind to provide the doc-string for the function.
The test TODO can be removed as they are provided by Kai already.